dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/csv/DefaultCsvEventService.java

```diff
@@ -48,28 +47,29 @@
 import com.fasterxml.jackson.dataformat.csv.CsvMapper;
 import com.fasterxml.jackson.dataformat.csv.CsvParser;
 import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+import com.google.common.collect.Lists;
```

Comment:
Do we prefer these over `java.util.List`?

Reply:
I don't think we have a policy on this.
I personally prefer this to initialise lists

Reply:
We are using it as part of our Java API. So not just to initialize a List. Since there is no policy I guess its a personal preference then 🙂 Would still be nice if it were consistent across all our Java APIs

Reply:
Sorry missed the `s` on the List 🤦🏻 so we are using java.util.List as part of our API :yum: